Tests/PointFreeTests/AboutTests.swift

```diff
@@ -17,16 +17,12 @@ class AboutTests: TestCase {
 
     #if !os(Linux)
     if #available(OSX 10.13, *), ProcessInfo.processInfo.environment["CIRCLECI"] == nil {
-      assertSnapshot(
-        of: .ioConnWebView(size: .init(width: 1080, height: 2300)),
-        matching: conn |> siteMiddleware,
-        named: "desktop"
-      )
-
-      assertSnapshot(
-        of: .ioConnWebView(size: .init(width: 400, height: 2300)),
-        matching: conn |> siteMiddleware,
-        named: "mobile"
+      assertSnapshots(
+        of: [
+          "desktop": .ioConnWebView(size: .init(width: 1080, height: 2300)),
+          "mobile": .ioConnWebView(size: .init(width: 400, height: 2300))
```

Comment:
A nice thing about baking this into the strategy instead is that it could maybe account for reusing the same loaded `webView` and merely resize it. We're currently taking on some extra work whenever we use `ioConnWebView`...